watchbuild/lib/watchbuild/runner.rb

```diff
@@ -44,13 +44,22 @@ def wait_for_build(start_time)
           Helper.log.error ex
           Helper.log.info "Something failed... trying again to recover"
         end
-        sleep 30
+
+        if WatchBuild.config[:sample_only_once].to_s == "false"
```

Comment:
Please don't do that, instead just `WatchBuild.config[:sample_only_once] == false` should work, right?